src/test/obj_pool_lock/obj_pool_lock.c

```diff
@@ -152,24 +107,31 @@ test_open_in_different_process(const char *path, int sleep)
 
 	pmemobj_close(pop);
 
-	UNLINK(path);
+	UNLINK(argv[1]);
 }
 #else
 static void
-test_open_in_different_process(const char *path, int sleep)
+test_open_in_different_process(char **argv, int argc, int sleep)
 {
 	PMEMobjpool *pop;
+	char *third_arg = "X";
```

Comment:
It took me a little while to figure out the meaning and purpose of this 'third argument' variable. Cryptic as hell ;)
If you decide to leave this little trickery here, I suggest adding a comment as to why.